extensions/setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+import sys
+
+if '..' not in sys.path:
+    sys.path.insert(0, '..')
+import setup as mypy_setup
+
+description = 'Experimental type system extensions for programs checked with the mypy typechecker.'
+long_description = '''
+Mypy Extensions
+===============
+
+The "mypy_extensions.typing" module defines experimental extensions to the
+standard "typing" module that are supported by the mypy typechecker.
+'''.lstrip()
+
+
+setup(
+    name='mypy_extensions',
+    version=mypy_setup.setup_kwargs['version'],
+    description=description,
+    long_description=long_description,
+    author=mypy_setup.setup_kwargs['author'],
+    author_email=mypy_setup.setup_kwargs['author_email'],
+    url=mypy_setup.setup_kwargs['url'],
+    license=mypy_setup.setup_kwargs['license'],
+    platforms=mypy_setup.setup_kwargs['platforms'],
+    package_dir={'mypy_extensions': 'mypy_extensions'},
+    py_modules=['mypy_extensions'],
+    packages=['mypy_extensions'],
+    scripts=[],
+    data_files=mypy_setup.find_data_files('mypy_extensions', ['*.py', '*.pyi']),
+    classifiers=mypy_setup.setup_kwargs['classifiers'],
+)
```

Comment:
This setup() call has all the same keys as the one in setup.py for mypy with one exception: There is no `cmdclass` key. I wasn't able to prove that it made sense here, so I did not include it.